src/warc2zim/content_rewriting/css.py

```diff
@@ -23,14 +36,26 @@ def rewrite(self, content: str | bytes) -> str:
             rules = parse_stylesheet(content)
         self.process_list(rules)
 
-        output = serialize(rules)
+        try:
+            output = serialize(rules)
+        except Exception:
+            fallback_rewriter = FallbackRegexCssRewriter(self.url_rewriter)
+            if isinstance(content, bytes):
```

Comment:
Is it possible to move this inside the generic RxRewriter so that it could benefit other rewriters? (I don't expect we will have many ways to convert a `bytes` into a `str` depending on the rewriter)